charts/htsget-rs/templates/ingress.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "htsget-rs.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" .Values.ingress.className}}
+  {{- end }}
+{{- end }}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1
+{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
There is absolutely no reason to support anything earler than `1.19`.
If you want this chart to be usable as is on any relevant version of k8s, ensure that you have correct separation between (>=1.19  <=1.24) and >=1.25.

Reply:
This should be done now.